target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -199,6 +199,9 @@ hc5861)
 	ucidef_set_led_netdev "wifi5g" "wifi5g" "$board:blue:wlan5g" "rai0"
 	ucidef_set_led_default "turbo" "turbo" "$board:blue:turbo" "0"
 	;;
+hc5962)
+	ucidef_set_led_timer "status" "status" "$board:white:status" "1000" "1000"
```

Comment:
It works like this:
![hc5962-led-status2](https://cloud.githubusercontent.com/assets/9377788/23303620/1e5bf2b2-fad0-11e6-8d71-2a1f9c60ddec.gif)